src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
```

Comment:
> The master delay can be used if you have a fast master sound card and a slow headphone sound card.

Is this a hypothetical use case or have you actually used a combination of sound cards with which you could perceive a difference in the output latency between them? If so, what were those sound cards? I'd imagine that in most cases the difference between output latencies between sound cards running at the same sample rate and buffer size would be very small.